components/library/http-parser/patches/02-gcc7.patch

```diff
@@ -0,0 +1,19 @@
+--- http-parser-2.6.2/http_parser.c.orig	2018-02-13 15:44:16.533837129 +0000
++++ http-parser-2.6.2/http_parser.c	2018-02-13 15:45:21.539702963 +0000
+@@ -1812,6 +1812,7 @@
+             case 0:
+               break;
+ 
++            /* FALLTHROUGH */
```

Comment:
Why? We have break here.

Reply:
Just imported upstream patch without thinking...